md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/AnalyticsAppAPI.java

```diff
@@ -0,0 +1,141 @@
+package com.wipro.ats.bdre.md.rest.ext;
+
+import com.wipro.ats.bdre.md.dao.ProcessDAO;
+import com.wipro.ats.bdre.md.dao.jpa.*;
+import com.wipro.ats.bdre.md.dao.jpa.Process;
+import com.wipro.ats.bdre.md.dao.jpa.Properties;
+import com.wipro.ats.bdre.md.rest.RestWrapper;
+import com.wipro.ats.bdre.md.rest.util.Dao2TableUtil;
+import com.wipro.ats.bdre.md.rest.util.DateConverter;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.servlet.http.HttpServletRequest;
+import java.security.Principal;
+import java.util.*;
+
+/**
+ * Created by SU324335 on 4/13/2016.
+ */
+@Controller
+@RequestMapping("/analyticsapp")
+public class AnalyticsAppAPI {
+    private static final Logger LOGGER = Logger.getLogger(AnalyticsAppAPI.class);
+    @Autowired
+    private ProcessDAO processDAO;
+
+    @RequestMapping(value = {"/createjobs"}, method = RequestMethod.POST)
+
+    @ResponseBody
+    public RestWrapper createJob(HttpServletRequest request,@RequestParam Map<String, String> map, Principal principal) {
+        LOGGER.debug(" value of map is " + map.size());
+        String appImagePathRequest = request.getParameter("appimage");
+        LOGGER.info("Path from request "+appImagePathRequest);
+
+        String actualPath = request.getRealPath(appImagePathRequest);
+        LOGGER.info("actual Path "+actualPath);
+        RestWrapper restWrapper = null;
+
+        String processName = null;
+        String processDescription = null;
+        Integer busDomainId = null;
+
+        Map<Process,List<Properties>> processPropertiesMap = new HashMap<Process, List<Properties>>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "processPropertiesMap" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)